### ISSUE TITLE
can't set minzoom=0

### ISSUE THREAD

**Issue by tyrasd** [mapbox/mapbox-tile-copy#68]
Setting minzoom to 0 doesn't produce any effect, presumably because the 0 is mistaken for a false somewhere like [here](https://github.com/mapbox/mapbox-tile-copy/blob/master/lib/tilelivecopy.js#L44) (the same applies to `--maxzoom=0`). Values > 0 work fine.

**Comment by mapsam** [comment]
> doesn't produce any effect

@tyrasd do you mean that there is no `0/0/0` tile generated here? I just tested with this geojson (I know, very unrealistic) and it tiles at `0/0/0` when minzoom is set to `0`. 

Something to keep in mind is that mapbox-tile-copy uses Mapnik when tiling particular data types - Mapnik makes some decisions about simplification that can lead to no geometry at z0, even if you specify a tile to exist at z0. The `mapbox-tile-copy` operations for `min/maxzoom` only say that the library should attempt to generate a tile here, if it results in an empty tile we throw it away.

**Comment by tyrasd** [comment]
Yes. Our use case was here: https://github.com/hotosm/osm-analytics-cruncher/blob/f2b71fc9c70e2279707e20f630a0402ce4aa2107/hotprojects.sh#L4, where we're uploading a large-ish geojson file as vector tiles to s3. Using minzoom=1 everything works fine, but with minzoom=0 results only get calculated up to zoom level 6 (if I remember correctly).

I'm pretty sure conditions like https://github.com/mapbox/mapbox-tile-copy/blob/master/bin/mapbox-tile-copy.js#L42 and https://github.com/mapbox/mapbox-tile-copy/blob/master/lib/tilelivecopy.js#L52 are causing issues when the supplied value is a literal numeric `0`.

**Comment by mapsam** [comment]
Ah, you're right! I tested with a case that also set `info.minzoom` to `0` so it didn't occur to me that `options.minzoom` was skipped. The following js snippets prove you are correct:

**bangbang**

```javascript
var minzoom = 0;
var maxzoom = 15;

[minzoom, maxzoom].forEach(function(zoom) {
  if (!!zoom) {
    console.log('zoom is set to ' + zoom);
  } else {
    console.log('zoom does not exist');
  }
});

// => zoom does not exist
// => zoom is set to 15
```

**check for defined value**

```javascript
var options = {
  minzoom: 0,
  maxzoom: 15
};

var minzoom = options.minzoom || 'there is no minzoom set';
var maxzoom = options.maxzoom || 'there is no maxzoom set';

console.log(minzoom); // => there is no minzoom set
console.log(maxzoom); // => 15
```

I think if we check for a numeric value here, i.e `minzoom >= 0` we should start fixing things. I'll start working on a fix here, thanks for the report @tyrasd!

**Comment by loganpowell** [comment]
Hi @tyrasd I'm trying to find a more streamlined workflow for setting min/maxzoom instead of using tippecanoe and ran into this issue. Did you ever find a solution?
